templates/formatter-examples.md.ejs

```diff
@@ -0,0 +1,57 @@
+---
+title: Documentation
+layout: doc
+---
+# ESLint Formatters
+
+ESLint comes with several built-in formatters to control the appearance of the linting results, and supports third-party formatters as well.
+
+The built-in formatter options are:
+
+<% Object.keys(formatterResults).forEach(function(formatterName) { -%>
+* [<%= formatterName %>](#<%= formatterName %>)
+<% }) -%>
+
+## Example Source
+
+Examples of each formatter were created from linting `fullOfProblems.js` using the `.eslintrc` configuration shown below.
+
+#### `fullOfProblems.js`
+
```

Comment:
OK, I was just trying to de-emphasize them, since the inline code tags give them a lot of visual weight as-is.